### PR TITLE
Improve plots legend

### PR DESCRIPTION
**Pull request by sashasashasasha151**
## Related issue(s)

https://issues.serokell.io/issue/EDNA-118

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Commit-and-PR-policy-4cf98e1a910a415d86b5f2491d9af1af).
- [ ] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

File: frontend/src/pages/dashboard/LegendInfo/LegendInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <div className={legendInfo("legend", { invisible: !legendActive })}>
+        {subExperiments.map(sex => (
+          <div key={sex.subexperiment.meta.id} className={legendInfo("legendElement")}>
+            <div>
+              <LegendSvg fill={sex.color} stroke={sex.color} />
+            </div>
+            <span>{`(${sex.subexperiment.target} ⟶ ${sex.subexperiment.compound})`}</span>
+            <div>
+              <span className={legendInfo("ic50")}>IC50:</span>
+              <span>{` ${sex.subexperiment.meta.item.result.Right[2]}`}</span>
```
Comment (gromakovsky) [inline frontend/src/pages/dashboard/LegendInfo/LegendInfo.tsx:42]: According to Figma the color of "IC50" string and the actual value should be identical:
image

**Comment by pva701** [comment]
image
I think you've spoiled compare mode: previously it showed points at the line (right?) but now it shows a description of each line, not point, and moreover it doesn't look like bearable to me.

**Comment by pva701** [comment]
ok, I see, points are not focused. If I scroll closer it works better.

**Comment by sashasashasasha151** [comment]
> ok, I see, points are not focused. If I scroll closer it works better.

What points?

File: frontend/src/pages/dashboard/DashboardPage.tsx [hunk 2]
```diff
@@ -106,11 +106,13 @@ interface PlotlyChartSuspendableProps {
 export function PlotlyChartSuspendable({
   className,
 }: PlotlyChartSuspendableProps): React.ReactElement {
-  const subExperiments = useRecoilValue(selectedSubExperimentsQuery).map(sex => ({
+  const subExperiments = useRecoilValue(selectedSubExperimentsExtraQuery).map(sex => ({
```
Comment (pva701) [inline frontend/src/pages/dashboard/DashboardPage.tsx:109]: maybe
```
...sex,
measurements: sex.measurements
      .slice()
      .sort((x, y) => x.item.concentration - y.item.concentration),
```
instead?

**Review by pva701 — APPROVED** [review]

**Pull request merged**